public-api/vx/note.php

```diff
@@ -58,16 +73,30 @@
 			if ( isset($_POST['tag']) )
 				$version_tag = coreSlugify_Name($_POST['tag']);
 
+			$parent = intval($_POST['parent']);
+
 			// Load Node
 			$node = node_GetById($node_id);
 
+
+			if ( isset($_POST['anonymous']) && $_POST['anonymous'] ) {
```

Comment:
I'm wondering if the note post should not be refused if it is set to be anon when the node is not allowing it. Looks to me like it would be posted non-anon then. This could happen if the node author changes the setting while you are composing your comment

Reply:
sorry now I see that's exactly how it works